apps/condo/domains/common/utils/pdf.ts

```diff
@@ -9,6 +9,7 @@ const PDF_FORMAT_SETTINGS = {
     // lineSpace - margin right for rest of lines (in css pixels)
     'a4': { pdfWidth: 210, pdfHeight: 297, elementOffset: 10, firstLineOffset: 23, lineSpace: 80 },
     'a5': { pdfWidth: 148, pdfHeight: 210, elementOffset: 10, firstLineOffset: 23, lineSpace: 80 },
+    'fullscreen': { pdfWidth: PDF_REPORT_WIDTH, pdfHeight: 297, elementOffset: 15, firstLineOffset: 23, lineSpace: 80 },
```

Comment:
I saw that we named this one `fullscreen` but below width is defined as 1920px. Maybe its worth to rename it to smth like `1080p`?

Reply:
Fixed